sdk/search/search-documents/samples/typescript/src/dataSourceConnections/getDataSourceConnection.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  SearchIndexerClient,
+  AzureKeyCredential,
+  SearchIndexerDataSourceConnection
+} from "@azure/search-documents";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint = process.env.SEARCH_API_ENDPOINT || "";
+const apiKey = process.env.SEARCH_API_KEY || "";
+
+async function main(): Promise<void> {
+  console.log(`Running Get Datasource Connection Sample....`);
+
+  const client = new SearchIndexerClient(endpoint, new AzureKeyCredential(apiKey));
```

Comment:
yeah like this one we have to do to update, so can't we combine?

Did .NET also do so many little samples?